### ISSUE TITLE
Drag and Drop Flow Issue

### ISSUE THREAD

**Issue by hollyschinsky** [phonegap/phonegap-app-desktop#185]
Long version
When I first open the GUI, I see the prompt to drop a folder in the center of the box and assume when I drag-n-drop a folder into it, the app will add/use what I dropped. Instead it opens the Create/Add menu and I'm not exactly sure what I'm supposed to do next. I'm dragging in an existing project assuming it's going to add it, not create so even showing an option for create seems weird to me but then I decide to click add since that's what I expected to happen when dragging in.  However, then when I click add, I get a file chooser asking me to select which project to add, which feels even more awkward because the path is not relative to anything I dragged over to begin with and I wonder what happened to the original folder I dragged over, and what was the point of it if I'm going to have to select it to be added anyway. 

TL;DR
Seems like the drag-in mechanism is really just a trigger to open the +/- menu to Create or Add and doesn't actually do anything with the folder you think you dropped in. I think it's obvious to click the +/- and would remove drag support unless it's fixed to actually add the folder you're dragging in without further prompting.

**Comment by bradleyflood** [comment]
Having this issue too...

**Comment by hermwong** [comment]
Fixed with: https://github.com/phonegap/phonegap-app-desktop/issues/678
